moviepy/audio/AudioClip.py

```diff
@@ -185,6 +185,19 @@ def write_audiofile(self, filename, fps=None, nbytes=2, buffersize=2000,
           
         logger
           Either 'bar' or None or any Proglog logger
+        
+        video (default None)
+          A video file path passed as input to ffmpeg to be copied into
+          the final output.
+          
+          ** NOTE: video options are intentionally limited. AudioClip is meant for
+                   working with audio. If you need advanced video functionality you
+                   should be using VideoClip or one of it's subclasses.
+        
+        video_codec (default 'copy')
```

Comment:
default "copy" but the actual default in the signature is `None`. It's a bit confusing. Could you clarify in the docstring?

Reply:
It looks like because None means copy per https://github.com/Zulko/moviepy/blob/d20acaaffb91bf46959ee3fc17be4cedb61eda29/moviepy/audio/io/ffmpeg_audiowriter.py#L58 but it could just as easily be 'copy'